components/input/docs/partials/api.md

```diff
@@ -296,11 +296,13 @@ Use the `format` attribute to set the format of the Inputmask.
 
 Default masking definitions:
 - 9 : numeric
-- a : alphabetical
+- a : alphabetic
 - \* : alphanumeric (letters and numbers)
```

Comment:
Could be good to ask some examples using these in this part of the doc. Not functional examples, just examples of the format string you would pass in for something like a phone number for example.